specs/darwin/fan_speeds.table

```diff
@@ -0,0 +1,11 @@
+table_name("fan_speeds")
+description("Fan speeds.")
+schema([
+    Column("fan", TEXT, "Fan number"),
```

Comment:
It's mostly up to you to define, then capture the "failed"/"unknown" state in the column description within the spec.